src/coreclr/tools/Common/TypeSystem/Interop/IL/MarshalHelpers.cs

```diff
@@ -556,7 +556,7 @@ public static partial class MarshalHelpers
                 if (nativeType == NativeTypeKind.AsAny)
                     return isAnsi ? MarshallerKind.AsAnyA : MarshallerKind.AsAnyW;
                 else
-                    return MarshallerKind.Invalid;
+                    return MarshallerKind.ComInterface;
```

Comment:
The default marshalling for objects is VARIANT. (See https://docs.microsoft.com/en-us/dotnet/framework/interop/default-marshaling-for-objects). Does this need more conditions?

Reply:
This was probably from my previous attempt to run over WinForms app. Do not actually used. I will remove.